plutus-pab/src/Cardano/Node/Server.hs

```diff
@@ -43,46 +44,43 @@ app serverHandler clientHandler stateVar =
         (healthcheck :<|> addTx :<|> getCurrentSlot :<|>
          (genRandomTx :<|>
           consumeEventHistory stateVar) :<|>
-          (newFollower :<|> getBlocks)
-          )
+          (newFollower :<|> getBlocks))
+
+data Ctx = Ctx { serverHandler :: Server.ServerHandler
+               , clientHandler :: Client.ClientHandler
+               , serverState   ::  MVar AppState
+               , clientState   :: MVar AppState
+               }
+
+main :: Trace IO MockServerLogMsg -> MockServerConfig -> Availability -> IO ()
+main trace MockServerConfig {..} availability = runLogEffects trace $ do
```

Comment:
I'd prefer to name the fields of `MockServerConfig` explicitly, without `RecordWildCards`. Makes it easier to read IMO.